tests/api/fixtures/populate.js

```diff
@@ -15,9 +15,9 @@ const API = module.exports = {
     return populatePromise
   },
 
-  createUserWithItems: async username => {
```

Comment:
I checked the code base and the only instance where this argument is used, it's an object, and an object is expected in `createUser` too, so I changed the name